src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCtorCall.java

```diff
@@ -79,13 +79,15 @@ public Valid() { //indent:4 exp:4
     } //indent:4 exp:4
 
     public Valid(InputIndentationCtorCall obj, long arg) { //indent:4 exp:4
-      obj.super( //indent:6 exp:6
-          arg); //indent:10 exp:10
+      obj //indent:6 exp:6
+          .super( //indent:10 exp:10
+            arg); //indent:12 exp:12
```

Comment:
You are changing existing tests that were probably testing something specific.
Can you create new constructors instead?

Reply:
Also I see no new test cases that produce a violation.

Reply:
Done.